### DISCUSSION TITLE
Can we avoid copying blob output from spamming the console?

### DISCUSSION THREAD

**Discussion by ssbarnea** [containers/podman#14295] [Ideas]
I am wondering if it would be possible to make the output lines such `Copying blob ...` which are spamming the terminal to end with `\r` instead of `\n` so they will not pile up. It would be more like a progressive more, user still being able to see that various files were downloaded but without taking precious terminal space.

It is even worse as many of these lines are also duplicated. Also using a quiet mode would likely has the downside of hiding useful information.

Using `\r` ok even with logging or redirecting to files, as in that case all the lines would still be visible.

**Comment by rhatdan** [comment]
This would be a very different experience that Docker, so I doubt we would do this.  --quiet is the only option.
